index.bs

```diff
@@ -744,7 +754,10 @@ To <dfn>populate the pose</dfn> of a {{XRSpace}} |sourceSpace| in an  {{XRSpace}
   1. Let |session| be |frame|'s {{XRFrame/session}} object.
   1. If |sourceSpace|'s [=XRSpace/session=] does not equal |session|, throw an {{InvalidStateError}} and abort these steps.
   1. If |destinationSpace|'s [=XRSpace/session=] does not equal |session|, throw an {{InvalidStateError}} and abort these steps.
-  1. If |destinationSpace|'s pose cannot be determined relative to |sourceSpace| at the time represented by |frame|, set |pose| to <code>null</code>.
+  1. If |destinationSpace|'s pose cannot be determined relative to |sourceSpace| at the time represented by |frame|, set |pose| to <code>null</code>, and abort these steps.
+  1. If either |sourceSpace| or |destinationSpace| have an [=XRSpace/internal type=] of {{XRReferenceSpaceType/identity}}, perform the following steps:
+      1. If the other space has a [=XRSpace/internal type=] that is [=XRSpace/input target ray=], and its [=XRSpace/input source=] has a {{XRInputSource/targetRayMode}} that is not {{XRTargetRayMode/screen}}, set |pose| to <code>null</code>, and abort these steps
+      1. If the other space has am [=XRSpace/internal type=] that is not [=XRSpace/viewer space=], set |pose| to <code>null</code>, and abort these steps
```

Comment:
One more minor change needed here: The flow of the logic indicates that target ray spaces would never resolve because their `internal type` is not both `input target ray` and `viewer space` at the same time.

I'd suggest something like the following:

1. If either |sourceSpace| or |destinationSpace| have an [=XRSpace/internal type=] of {{XRReferenceSpaceType/identity}}, perform the following steps:
      1. If the other space has an [=XRSpace/internal type=] that is not [=XRSpace/viewer space=] or [=XRSpace/input target ray=], set |pose| to <code>null</code>, and abort these steps.
      1. If the other space has an [=XRSpace/internal type=] of [=XRSpace/input target ray=] and the [=XRSpace/input source=] has a {{XRInputSource/targetRayMode}} that is not {{XRTargetRayMode/screen}}, set |pose| to <code>null</code>, and abort these steps.

Reply:
Ah, gotcha, I didn't pay close attention when fixing this, sorry.

Done!